base_geoengine/geo_field.py

```diff
@@ -54,32 +56,31 @@ def set_geo(self, value):
             return None
         return res.wkt
 
-
-
     _type = None
     _classic_read = False
     _classic_write = True
     _symbol_c = u' %s'
-    #Due to the conception of the orm we have to set symbol set as an instance variable
-    # instead of a class variable
-    #_symbol_set = (' ST_GeomFromText(%s)', set_geo)
+    # Due to the conception of the orm we have to set symbol set as an instance
+    # variable instead of a class variable_symbol_set = (' ST_GeomFromText(%s)'
+    # , set_geo)
     _symbol_get = load_geo
     _fnct_inv = True
 
+    # geometry type of postgis point, multipolygon, etc...
+    _geo_type = None
 
-    def __init__(self, string, geo_type, dim=2, srid=900913 , gist_index=True, **args):
+    def __init__(self, string, dim=2, srid=900913, gist_index=True,
+                 **args):
```

Comment:
This should be fixed on the 6.1 and then forward ported into the others branches